stock_by_warehouse_mrp_bom/README.rst

```diff
@@ -0,0 +1,42 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+    :alt: License: LGPL-3
+
+Functional Rationale:
+---------------------
+
+Provide a nice widget in the product on the bill of material's line to see the quantity of product
+by warehouse in order to help you to review such information as fast as possible.
+
+.. image:: ./static/description/widget_by_warehouse.png
```

Comment:
The image in here should be the one in the BOM line and not the one in the product